src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double upperSide;
+    private final double bottomSide;
+    private final double height;
+
+    public IsoscelesTrapezoid(String color, double upperSide, double bottomSide, double height) {
+        super(color);
+        this.upperSide = upperSide;
+        this.bottomSide = bottomSide;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return ((bottomSide + upperSide) / 2) * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: isosceles trapezoid," + " area: "
+                + getArea() + " sq. units, bottom side: "
+                + bottomSide + " units, upper side: "
+                + upperSide + " units, height: "
+                + height + " units, color: " + super.getColor());
```

Comment:
Use English in the messages within the `draw()` method. The word 'isosceles' is spelled correctly, but ensure that all other messages are also in English and free of typos for consistency and universality.